pom.xml

```diff
@@ -198,7 +239,7 @@
                         {
                         "functionName": "onebusaway-alexa",
                         "description": "OneBusAway Alexa skill",
-                        "handler": "org.onebusaway.alexa.LambdaFunctionHandler",
+                        "handler": "org.onebusaway.alexa.OneBusAwaySkill",

```

Comment:
Is this correct? After deploying to a dev environment, the only way I could get the skill to work is by configuring the Lambda console to use the handler `org.onebusaway.alexa.OneBusAwayAlexa`. 

If I configured it to `org.onebusaway.alexa.OneBusAwaySkill`, it says:

~~~
Class not found: org.onebusaway.alexa.OneBusAwaySkill: java.lang.ClassNotFoundException
~~~

Also, within the class `org.onebusaway.alexa.OneBusAwayAlexa`, it says:

~~~
private static Skill getSkill() {
        return SpringContext.getInstance().getBean("oneBusAwaySkill", Skill.class);
    }
~~~

Does it matter what this string is?

Reply:
* Good catch, the correct name for deployment should be `OneBusAwayAlexa `, because the class name is `OneBusAwayAlexa`, and the `oneBusAwaySkill` string matters because in `ApplicationConfig.java`, the bean name is `oneBusAwaySkill`.

Reply:
I will update the readme to address it